extension/src/paymentHandler/manual-capture.handler.js

```diff
@@ -2,26 +2,22 @@ const pU = require('./payment-utils')
 const { manualCapture } = require('../service/web-component-service')
 const {
   CTP_INTERACTION_TYPE_MANUAL_CAPTURE,
-  CTP_CUSTOM_FIELD_MANUAL_CAPTURE_RESPONSE
 } = require('../config/constants')
 
 async function execute (paymentObject) {
-  const manualCaptureRequestObj = JSON.parse(paymentObject.custom.fields.manualCaptureRequest)
-  const { request, response } = await manualCapture(manualCaptureRequestObj)
+  const chargeInitialTransaction = pU.getChargeTransactionInitial(paymentObject)
+  const authorizationSuccessTransaction = pU.getAuthorizationTransactionSuccess(paymentObject)
+  const { request, response } = await manualCapture(chargeInitialTransaction,
```

Comment:
actually interaction id for add transaction is optional, maybe we should not add it in the first place, WDYT?

Reply:
The interaction ID is quite useful. It helps you to connect with the interfaceInteraction. So I would add it there.

Reply:
OK, I check again, it's different than what I thought.